lib/application/coordinator/routes.dart

```diff
@@ -87,3 +87,16 @@ class SettingsPath extends AppPath {
   @override
   String get formattedPath => '/$name';
 }
+
+//
+// Update Collection
+//
+class UpdateCollectionPath extends AppPath {
```

Comment:
Isn't it odd to name it "update" but it's actually creation and update at the same time?

Reply:
That's the pattern we've followed in other projects to name things that were creation and edition at the same time. We could also use `Set` but it seems stranger than `update` to me.